lib/improver/tests/nowcasting/optical_flow/test_OpticalFlow.py

```diff
@@ -654,19 +654,19 @@ def test_values(self):
         self.assertAlmostEqual(np.mean(vcube.data), 2.1719084)
 
     def test_update_smoothing_radius(self):
```

Comment:
I think we should have another test where the time delta reduces the time interval between the cubes, and the radius stays the same and the test produces output.